backend/long_running_request_actors.py

```diff
@@ -72,3 +72,11 @@ def auto_assign_reviewers():
     )
 
     work()
+
```

Comment:
There is another dramatiq actor in the same file that looks like this:

```
@dramatiq.actor()
def recruit_project_setup_repository(project_id):
    from curriculum_tracking.models import RecruitProject
    project = RecruitProject.objects.get(pk=project_id)
    project.setup_repository()

```

I'm sure you can see that this code is not DRY. Why not make use of the existing actor? Juist pass it the project id instead of the card id.